Current day button

jdial1 opened the pull request:
![currentday](https://user-images.githubusercontent.com/486523/61712938-0a4d3680-ad1d-11e9-8044-97868e924e1b.jpg)

jdial1 committed:
Added alerts to Sign up and Sign In page

jdial1 committed:
gradle upgrade

jdial1 committed:
Update sign_up.dart

jdial1 committed:
Update sign_in.dart

jdial1 committed:
Update sign_up.dart

jdial1 committed:
Added logicffor Curve Colors

jdial1 committed:
Merge branch 'master' of https://github.com/jdial1/Colorie

jdial1 committed:
Locked Orientation

jdial1 committed:
Added Current date to AppBar

jdial1 committed:
Reset to Elven Master

jdial1 committed:
reset to master

jdial1 committed:
Added Current Day Button

10ndavis commented:
Idk how my comment got deleted. But let's change the function name to something like setToCurrentDay and use widget.today from above instead of a new datetime.

jdial1 committed:
Added Suggested Corrections

Function name changed to SetToCurrentDay

Changed DateTime.now() to widget.today

jdial1 commented:
Added Corrections
